invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -110,7 +110,7 @@ class CommunityProfileForm extends Component {
       access: {
         visibility: "public",
         member_policy: "open",
-        record_policy: "open",
+        submission_policy: "open",
```

Comment:
I think all the names should be "record_submission_policy" to avoid confusion with "submission_review_policy" (as discussed on the demo.
Maybe others would have also some suggestions on the naming, to find the pair which is the least confusing ping @slint @ptamarit @sakshamarora1 @yashlamba @0einstein0

Reply:
yeah `record_submission_policy` sounds more befitting or maybe `community_inclusion_policy` since they are community inclusion requests?

Reply:
I think this comment hasn't been addressed regarding renaming